### PR TITLE
Added the sandwich recipe

### PR DESCRIPTION
**Pull request by chlotastrophie**
#5

### CODE REVIEW

File: sandwich.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Ingredients: bread, mayonnaise, cheese, and ham
```
Comment (noahsark769) [inline sandwich.md]: Can we please put these on separate lines?

File: sandwich.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Instructions: 1. Tell your gf to make you a sandwich 2. Get 2 slices of bread 3. Spread mayonnaise on one or both sides depending on preference 4. Put however many slices of cheese 5. Put however many slices of ham 6. Put bread on top 7. Have her give you sandwich 8. Done
```
Comment (noahsark769) [inline sandwich.md]: These too :)

**Comment by chlotastrophie** [comment]
@noahsark769 fixed it

**Comment by noahsark769** [comment]
GREAT

**Pull request merged**